report/slack.go

```diff
@@ -21,44 +21,45 @@ const (
 // SlackReporter is the reporter to send the profiling report to the
 // specific Slack channel.
 type SlackReporter struct {
-	app     string
-	channel string
+	app       string
+	channelID string
```

Comment:
How about leaving the `channel` parameter and commenting on the deprecation with the official deprecation link? And, call the new Slack API by checking whether the `channelD` is set, or call the legacy API if not.

This can prevent breaking changes and induce users to update their codes.

Reply:
this sounds better.
UploadFileContext is deprecated in the latest slack package, not gone.

Reply:
I like this idea too! I’ll go with this approach

Reply:
[00db77c](https://github.com/daangn/autopprof/pull/18/commits/00db77cef9aa51d786b6cf538afde8d6671210d8)
I've Updated params to provide both `Channel` and `ChannelID` and added comments.

Reply:
I’ve added a private function that calls the latest API based on whether ChannelID is set, so please take a look